### ISSUE TITLE
Merge clears & takes into single chart / table

### ISSUE THREAD

**Issue by mattyg** [rainlanguage/rain.orderbook#235]
Use client-side pagination to merge clears & takes into single chart / table.

Rename chart / table titles to "trades" or "swaps" on the order page

**Comment by hardyjosh** [comment]
@mattyg is this one needed anymore? We're removing clears and I already renamed the chart titles to "trades" everywhere

**Comment by mattyg** [comment]
Yes I think we want to include clears data merged in the same list/chat as takes as labeled as "trades". Removing clears as standalone is temporary until we get to implement that.

**Comment by thedavidmeister** [comment]
🤖 ai:producer
Re-verified against main 2026-07-07: appears RESOLVED — the order page now renders a single unified Trades list: packages/ui-components/src/lib/components/tables/OrderTradesListTable.svelte, mounted from components/detail/OrderDetail.svelte. The separate clears/takes tables belonged to the removed tauri app. Queued for human close review.
